kubernetes/samples/scripts/domain-lifecycle/helper.sh

```diff
@@ -266,6 +282,21 @@ createPatchJsonToUpdateReplicaAndPolicy() {
   eval $__result="'${patchJson}'"
 }
 
+#
+# Function to create patch json string to update replica and policy
```

Comment:
This says both replica and startPolicy but the patch string doesn't include replica

Reply:
Corrected